2-mandatory/1-weather-report.js

```diff
@@ -11,53 +11,83 @@
         - Hint: you can call the temperatureService function from your function
 */
 
-function getTemperatureReport(cities) {
-    // TODO
+let cities = [
+  "London",
+  "Paris",
+  "Barcelona",
+  "Dubai",
+  "Mumbai",
+  "São Paulo",
+  "Lagos",
+];
+let citiesTemp = [
+  "London",
+  10,
+  "Paris",
+  12,
+  "Barcelona",
+  17,
+  "Dubai",
+  27,
+  "Mumbai",
+  29,
+  "São Paulo",
+  23,
+  "Lagos",
+  33,
+];
+
+function getTempratur(city) {
+  let index = citiesTemp.indexOf(city);
+  index++;
+  return citiesTemp[index];
 }
```

Comment:
This is an interesting approach! I think in some lower-level programming languages, storing info like this (an array with alternating keys and values) might be more common place. In JS, it's more conventional to use a dictionary:

```js
const cityTemperatures = {
    "São Paulo":  23,
  "Lagos":  33
}
```

or a `Map` as used by the function the exercise provides below (the exercise suggested you use that function rather than write your own, but nothing wrong with trying things out for yourself :muscle: )